torch/_subclasses/fake_tensor.py

```diff
@@ -1157,7 +1157,7 @@ def dispatch(self, func, types, args=(), kwargs=None):
 
             return converter(self, args[0])
 
-        args, kwargs = self.validate_and_convert_non_fake_tensors(
+        args, kwargs, flat_arg_fake_tensors = self.validate_and_convert_non_fake_tensors(
```

Comment:
Can you add back the comment you had earlier ?

Reply:
done